tests/components/mqtt/test_init.py

```diff
@@ -2145,10 +2121,9 @@ async def test_setup_manual_mqtt_with_platform_key(
 
 
 @pytest.mark.parametrize("hass_config", [{mqtt.DOMAIN: {"light": {"name": "test"}}}])
+@pytest.mark.usefixtures("hass")
```

Comment:
Are these needed?
I've usually found that hass can be dropped if unused.

Reply:
I'll check

Reply:
Seems they are not needed, so I removed those. Thanks!